tests/fermi/test_bravyi_kitaev.py

```diff
@@ -995,6 +996,22 @@ def test_fermi_sentence_identity():
             }
         ),
     ),
+    (
```

Comment:
How come none of the other tests were hitting this bug? 🤔

Reply:
Apparently, BK and Parity gave the same results for them. I will put a TODO to make the FermiSentence tests as strong as the FermiWord ones.